sentry_sdk/utils.py

```diff
@@ -358,7 +358,10 @@ def safe_repr(value):
 
 
 def filename_for_module(module, abs_path):
-    # type: (str, str) -> str
+    # type: (str, Optional[str]) -> Optional[str]
```

Comment:
Looking at `serialize_frame`, `module` can be `None` too.

Reply:
You're right, not sure why mypy didn't catch that one. I think I fixed it now though!